features/tests/test_api.py

```diff
@@ -341,6 +343,34 @@ def test_feature_opening_hours(snapshot, api_client):
     snapshot.assert_match(executed)
 
 
+def test_price_list(snapshot, api_client):
+    PriceTagFactory(price=Decimal("100.01"), unit="a month")
+    PriceTagFactory(price=Decimal("200.01"), unit="a year")
+
+    executed = api_client.execute(
+        """
+    query FeaturePriceList {
+          features {
+            edges {
+              node {
+                properties {
+                  details {
+                    priceList {
+                      item
+                      price
+                      unit
+                    }
+                  }
+                  }
+                }
+              }
+            }
+          }
+    """
```

Comment:
yeah, sorry for not mentioning it right in the message, but I my OCD expects this format 😅 
```python
"""
query FeaturePriceList {
    features {
        edges {
            node {
                properties {
                    details {
                        priceList {
                            item
                            price
                            unit
                        }
                    }
                }
            }
        }
    }
}
"""
```

Reply:
ok, it seems that Juha used 2 spaces for query string like this, I will update my suggestions

Reply:
go for it! =)

Reply:
updated first comment in this thread. I used 4 spaces for indent initially, now it's 2

Reply:
updated with your suggestion :) thanks!